conf/postgres-operator/cluster-deployment.json

```diff
@@ -0,0 +1,360 @@
+{
```

Comment:
I think this file needs to be removed?

Reply:
Agreed, it looks like it was added back during the most recent rebase. Removed, thanks.